hw3/01-characters.js

```diff
@@ -1,4 +1,32 @@
 // url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters";
+const charactersElement = document.getElementById("characters");
 
-fetch(url);
+fetch(url)
+  .then((response) => response.json())
+  .then((characters) => {
+    characters.forEach((character) => {
+      const characterElement = document.createElement("div");
+      characterElement.classList.add("character");
+
+      const characterImage = document.createElement("img");
+      characterImage.src = character.imageUrl;
+      characterImage.alt = `${character.fullName} image`;
```

Comment:
In the alt text it is not necessary to add words/phrases such as "... image", “image of” or “graphic of”.